### PR TITLE
Simply Auth

### PR DESCRIPTION
**Pull request by ChenYFan**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,10 +2,12 @@
...
...background-color:rgba(255,255,255,.3)}.bg-bubbles li:nth-child(10){left:90%;width:160px;height:160px;animation-delay:11s}@-webkit-keyframes square{0%{transform:translatey(0)}100%{transform:translatey(-700px) rotate(600deg)}}@keyframes square{0%{transform:translatey(0)}100%{transform:translatey(-700px) rotate(600deg)}}@media (max-width:100px){.container h1{font-size:20px}form button,form input{width:100px}}</style> </head><body><div id="all"><div class="wrapper"><div class="bg-container"><div class="container"><h1 style="margin: 0;" id="bar">Welcome</h1><form class="form" id="fm"><input id="username" type="text" placeholder="用户名" value name="username" /><input id="password" type="password" placeholder="密码" value name="password" /><button type="button" id="login-button">登陆</button><br /><br /></form></div></div><ul class="bg-bubbles"><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li><li></li></ul></div></div><script>  $("#login-button").click(function (event) {   document.cookie="username=" + document.getElementById("username").value;  document.cookie="password="+document.getElementById("password").value;  location.reload();  });</script></body></html>`
```
Comment (TkzcM) [inline index.js:10]: zh-cmn-Hans is deprecated, use zh-Hans instead.
**DON'T save username / password as plaintext.**

**Comment by TkzcM** [comment]
Thanks for your contribution!
But I decided to implement this feature by myself.
Sorry.

**Comment by ChenYFan** [comment]
Well, I just saw this project and thought that it was not enough without verification so I add a simply auth.
Anyway,are you Chinese?I think that who use BaiduWangPan(or BaiduNetworkDisk) is usually Chinese.

File: index.js (hunk 1 above)
Comment (ChenYFan) [reply index.js:10]: About zh-cmn-Hans,It is taking from my old project and I never pay attention to that.
> NEVER store username / password as plaintext.
Yes,but I think taking it in HTTPS is safe.In my old project,I force the use of HTTPS so I didn't care about that.

**Comment by TkzcM** [comment]
> Anyway,are you Chinese?I think that who use BaiduWangPan(or BaiduNetworkDisk) is usually Chinese.

Yes. Although now I'm living in Japan.

**Pull request closed without merging**